### PR TITLE
Able to work on colab almost effectively except some functionalities

### PR DESCRIPTION
**Pull request by alihussainia**
I have updated the widget.py file and based on my modifications, the altair_widgets are running on colab and plus I was able to run a demo colab notebook in which I have shown the working of the altair_widgets with a local customized package based on this pull request widget.py.[Demo Colab Link](https://github.com/alihussainia/altair_widgets/blob/master/altair_widgets_demo.ipynb)

### CODE REVIEW

File: altair_widgets/widget.py [hunk 1]
```diff
@@ -1,9 +1,26 @@
+from subprocess import call
+
+call("jupyter nbextension enable --py --sys-prefix widgetsnbextension", shell=True)
+call("jupyter nbextension enable --py --sys-prefix vega", shell=True)
+call("pip install altair_saver", shell=True)
+call("pip install jupyter pandas vega", shell=True)
+call("pip install --upgrade notebook", shell=True)
+call("jupyter nbextension install --sys-prefix --py vega", shell=True)
+call("apt-get update", shell=True)
+call("apt-get -qq install chromium-chromedriver", shell=True)
```
Comment (stsievert) [inline altair_widgets/widget.py:10]: I think a better place for this code would be in the README (this code shouldn't run everytime someone types `import altair_widgets`, and this code may not work for everyone not everyone (e.g., if they don't have the `apt-get` command)).

**Review by stsievert — COMMENTED** [review]
Thanks for this PR!

**Comment by stsievert** [comment]
@alihussainia I've added the commands to the README, and removed other files. Does that look good to you?

**Comment by alihussainia** [comment]
Thank you, brother @stsievert it is fine for me but I would say that what if the commands are added to the widget.py file as if you test the widget.py file along with these commands on regular jupyter notebooks and if they won't give any error then it will be good addition to the widget.py especially for those who are looking forward to use it on colab by directly importing the project via pip :)

**Comment by stsievert** [comment]
I've made some changes; now, I issue a warning if the notebook is running on colab, and point to the docs. Would that address the issue?

**Comment by alihussainia** [comment]
@stsievert awesome amigo! Thanks bro. I thought I would also become a contributor to the repo but its fine :) thanks for your time and consideration. 

Best wishes,

Muhammad Ali

**Comment by alihussainia** [comment]
Should I close the PR?

**Comment by stsievert** [comment]
Perfect, I've merged this PR into 20f66597532626a17456f58a3bc3e3a9d4d24c82 (which we co-authored).

**Comment by alihussainia** [comment]
Thank you, Thank you so much brother! You just made my day amigo. I am truly happy and grateful to you for this favor brother. Please share your linkedin id too, I would definitely like to connect with such an awesome brother of mine. 

Thanks once again brother.

Muhammad Ali
Your brother and now a contributor too <3

**Pull request merged**